e2e/hook_test.go

```diff
@@ -112,7 +112,7 @@ func testHook() {
 		stdout, stderr, err := kubectlWithInput(hookGenericEphemeralVolumeYAML, "-n", nsHookTest, "apply", "-f", "-")
 		Expect(err).ShouldNot(HaveOccurred(), "stdout=%s, stderr=%s", stdout, stderr)
 
-		By("checking pod is annotated with topolvm.io/capacity")
+		By("checking pod is annotated with" + topolvm.GetPluginName() + "/capacity")
```

Comment:
Because the return type of `GetCapacityResource()` is `corev1.ResourceName`, not string, we need a type conversion like:
```
By("checking pod is annotated with" + string(topolvm.GetCapacityResource()))
```

Is that you want? Or should I define another function to return string type value?

Reply:
core/v1 implements a dedicated method that returns its name in string.
https://pkg.go.dev/k8s.io/api/core/v1#ResourceName.String

Reply:
Fixed.